# This file is part of HiGFXback

# requires
REQUIRES="autotools-wrappers-am-build"

pkg-config --exists --print-errors $REQUIRES || exit 1

# configure (ac-2.64; am-1.11)
aclocal -I m4; autoheader; autoconf; automake -a -c
./configure --prefix=

# build
make

# install
install -d $DESTDIR/bin
install gunzip $DESTDIR/bin
ln -sf gunzip $DESTDIR/bin/uncompress
install gzexe $DESTDIR/bin
install gzip $DESTDIR/bin
install zcat $DESTDIR/bin
install zcmp $DESTDIR/bin
install zdiff $DESTDIR/bin
install zegrep $DESTDIR/bin
install zfgrep $DESTDIR/bin
install zforce $DESTDIR/bin
install zgrep $DESTDIR/bin
install zless $DESTDIR/bin
install zmore $DESTDIR/bin
install znew $DESTDIR/bin

# build.pc
install -d $DESTDIR/share/pkgconfig
cat > $DESTDIR/share/pkgconfig/gzip-build.pc << EOF
Name: gzip
Version: 1.3.13
Description: GNU zip file format compressor / decompressor
Requires: $REQUIRES

exec=\\
/bin/gunzip \\
/bin/gzexe \\
/bin/gzip \\
/bin/uncompress \\
/bin/zcat \\
/bin/zcmp \\
/bin/zdiff \\
/bin/zegrep \\
/bin/zfgrep \\
/bin/zforce \\
/bin/zgrep \\
/bin/zless \\
/bin/zmore \\
/bin/znew
EOF

exit

# patch
--- gzip-1.3.13.orig/configure.ac
+++ gzip-1.3.13/configure.ac
@@ -29,6 +29,7 @@
 AC_CONFIG_HEADERS([lib/config.h:lib/config.hin])
 AM_INIT_AUTOMAKE([1.11 dist-xz])
 AM_SILENT_RULES([yes]) # make --enable-silent-rules the default.
+AM_MAINTAINER_MODE
 
 AC_PROG_CC_STDC
 AM_PROG_CC_C_O

# source
https://ftp.gnu.org/gnu/gzip/gzip-1.3.13.tar.gz
